static/app/components/feedback/list/issueTrackingSignalsWrapper.tsx

```diff
@@ -0,0 +1,20 @@
+import IssueTrackingSignals from 'sentry/components/feedback/list/issueTrackingSignals';
+import useFetchFeedbackData from 'sentry/components/feedback/useFetchFeedbackData';
+import {Group} from 'sentry/types';
+import {FeedbackIssue} from 'sentry/utils/feedback/types';
+
+interface Props {
+  feedbackIssue: FeedbackIssue;
+}
+
+export default function IssueTrackingSignalsWrapper({feedbackIssue}: Props) {
+  const {issueData, eventData} = useFetchFeedbackData({feedbackId: feedbackIssue.id});
```

Comment:
unfortunately need to refetch issue data here because the infinite list returns an incomplete issue (e.g. `pluginActions` etc don't exist which is needed to fetch the plugins available) :(

Reply:
What if we were to add support for `?expand=actions` (or some name like that) to the list endpoint?

I think we'd need to do a few codechanges to the api, but nothing crazy.
1. Make this `_get_actions` method available outside this class, maybe put it into `src/sentry/models/groupinbox.py` https://github.com/getsentry/sentry/blob/bb1cbec061c44b077ed57e103d7da4f3fc937e59/src/sentry/api/endpoints/group_details.py#L81-L106
2. The `OrganizationGroupIndexEndpoint` class uses `StreamGroupSerializerSnuba`, so we could add a new if-statement for our new "expand" right here: https://github.com/getsentry/sentry/blob/bb1cbec061c44b077ed57e103d7da4f3fc937e59/src/sentry/api/serializers/models/group_stream.py#L297-L307

cc @JoshFerge